### DISCUSSION TITLE
Deploy from another directory?

### DISCUSSION THREAD

**Discussion by snelgrove** [beginner-corp/begin-community#73] [Q&A] [answered]
I’m trying TypeScript in a Begin project. My brain won’t allow the compiled .js files to sit amongst the .ts source files, so I’d like to tell begin to deploy from the /dist directory. Is this possible?

My current workaround to to put the source in /src.ts and output the compiled JavaScript to /src. I don’t know how long I can allow that either 😬

**Comment by snelgrove** [comment, marked as answer]
I haven't tried it yet, but it sounds like this is the way to serve transpiled code from another directory: 

[Developer Experience > Custom Source Paths](https://arc.codes/docs/en/guides/developer-experience/custom-source-paths)

Seems like a neat approach would be to use the verbose syntax to serve routes, events, etc out of `./dist`

**Reply by brianleroux** [reply]
Yes this should work for you. Also you can change the static folder directive to deploy from another directory for frontend assets.
